### PR TITLE
use oidc endpoint for issuer validation. 

### PR DESCRIPTION
**Pull request by jennyf19**
#1167 + css related issue

### CODE REVIEW

File: src/Microsoft.Identity.Web/Resource/MicrosoftIdentityIssuerValidator.cs [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+            return string.Empty;
+        }
+
+        // The AAD "iss" claims contains the tenant ID in its value.
+        // The URI can be
+        // - {domain}/{tid}/v2.0
+        // - {domain}/{tid}/v2.0/
+        // - {domain}/{tfp}/{tid}/{userFlow}/v2.0/
+        private static string GetTenantIdFromIss(string iss)
```
Comment (hpsin) [inline src/Microsoft.Identity.Web/Resource/MicrosoftIdentityIssuerValidator.cs:227]: Will this run on non-AAD tokens? I.e. standard oidc tokens?

File: src/Microsoft.Identity.Web/Resource/AadIssuerValidator.cs [hunk 2]
```diff
@@ -192,16 +245,5 @@ private static string GetTenantIdFromIss(string iss)
 
             return string.Empty;
         }
-
-        /// <summary>
-        /// This method is now Obsolete.
-        /// </summary>
-        /// <param name="aadAuthority">Aad authority.</param>
-        /// <returns>NotImplementedException.</returns>
-        [Obsolete(IDWebErrorMessage.AadIssuerValidatorGetIssuerValidatorIsObsolete, true)]
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web/Resource/AadIssuerValidator.cs:201]: I guess it's ok to remove this obsolete method.

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @jennyf19 

It's a great simplification (and will be more resilient to clouds)

File: src/Microsoft.Identity.Web/Resource/AadIssuerValidator.cs (hunk 2 above)
Comment (jennyf19) [reply src/Microsoft.Identity.Web/Resource/AadIssuerValidator.cs:201]: can add it back if you want.

**Pull request merged**